Xamarin.Forms.Platform.UAP/CollectionView/GroupedItemTemplateCollection.cs

```diff
@@ -51,26 +54,36 @@ GroupTemplateContext CreateGroupTemplateContext(object group)
 
 		void GroupsChanged(object sender, NotifyCollectionChangedEventArgs args)
 		{
-			switch (args.Action)
+			if (SynchronizationContext.Current != _synchronizationContext)
+				_synchronizationContext.Post(RaiseCollectionChanged, args);
```

Comment:
```suggestion
			{
				_synchronizationContext.Post(RaiseCollectionChanged, args);
			}
```

Maybe it's my personal preference, but can we have brackets in here? Seems a bit more readable to me with the `else` in there. Also, more in line with the other code that has this change

Reply:
Thanks for the feedback. The code has changed a bit and this change is no longer necessary.